service/api/types.go

```diff
@@ -42,6 +42,10 @@ type Breakpoint struct {
 	Goroutine bool `json:"goroutine"`
 	// variables to evaluate
 	Variables []string `json:"variables,omitempty"`
+	// number of times a breakpoint has been reached in a certain goroutine
+	HitCount map[string]uint64 `json:"hitCount"`
```

Comment:
Unfortunately when converting to json, if `HitCount` would be `map[int]uint64` then marshaling will not work properly.
See the exemple here: https://play.golang.org/p/-1EMYNSCoa vs the working version: https://play.golang.org/p/MscIV7i306